lib/evss/gi_bill_status/availability.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/service_status'
+require 'evss/response'
+
+module EVSS
+  module GiBillStatus
+    class Availability < EVSS::Response
```

Comment:
as it's not a response from an API I think this could just be a `Common::Base` model, although even that has pagination/sorting methods, the bare minimum would be a Virtus model (which gives you the auto named arg) and serialization:
```ruby
module EVSS
  module GiBillStatus
    class Availability
      include ActiveModel::Serialization
      include Virtus.model(nullify_blank: true)
```

Reply:
I'll give that a shot.  I was just desperately trying to figure out what I needed to inherit from / mixin, in order to get the `data: {attributes: {}}` wrapping of the response json.

Seems to me, that this wrapping should have it's own dedicated mixin so I don't have to include a whole bunch of extra stuff just to get that wrapping functionality.